decoder/DecoderApi_unittest.cpp

```diff
@@ -297,7 +311,14 @@ TEST_P(DecodeApiTest, Format_Change)
             allocator->onFormatChange(format);
 
             //send buffer again
-            EXPECT_EQ(YAMI_SUCCESS, decoder->decode(&buffer));
+            status = decoder->decode(&buffer);
+            if (YAMI_UNSUPPORTED == status) {
```

Comment:
hi @xuguangxin ,
This is necessary actually. Because Yami will return YAMI_DECODE_FORMAT_CHANGE when decoding the first frame.